internal/api/grpc/user/v2beta/query.go

```diff
@@ -33,7 +33,7 @@ func (s *Server) ListUsers(ctx context.Context, req *user.ListUsersRequest) (*us
 	if err != nil {
 		return nil, err
 	}
-	res, err := s.query.SearchUsers(ctx, queries, s.checkPermission)
+	res, err := s.query.SearchUsers(ctx, queries, "", s.checkPermission)
```

Comment:
This API is (more or less) a copy of `user/v2`. We need to have the same change here as you did there.